casbin/internal_enforcer.py

```diff
@@ -21,7 +19,22 @@ def _add_policy(self, sec, ptype, rule):
                 self.watcher.update()
 
         return rule_added
+    
+    def _add_policies(self,sec,ptype,rules):
+        """adds rules to the current policy."""
+        rules_added = self.model.add_policies(sec, ptype, rules)
+        if not rules_added:
+            return rules_added
+
+        if self.adapter and self.auto_save:
+            if self.adapter.add_policies(sec, ptype, rules) is False:
+                return False
```

Comment:
Here you need to determine whether it is a `BatchAdapter`, the same in `_remove_policies()`.

Reply:
@yyellowsun

Reply:
@techoner  I fixed it,please review again.